bottles/frontend/ui/env-var-entry.blp

```diff
@@ -2,12 +2,12 @@ using Gtk 4.0;
 using Adw 1;
 
 template $EnvVarEntry: Adw.EntryRow {
```

Comment:
This should be renamed to `BottlesEnvironmentVariableEntry`.

Reply:
Done

Reply:
Do you think we should also rename the files to longer names? Currently, they are `dialog-env-vars.blp` and `env-var-entry.blp`.

Reply:
Yeah... I don't really know how to approach it. I do agree that there should be a standard that makes more sense. I only follow that convention on things I recently rework on